modules/oe_content_publication/config/install/field.field.node.oe_publication.oe_documents.yml

```diff
@@ -9,7 +9,7 @@ id: node.oe_publication.oe_documents
 field_name: oe_documents
 entity_type: node
 bundle: oe_publication
-label: File
+label: Files
```

Comment:
Configuration has to updated on existing sites through post_update

Reply:
This is done.